Google/resource-manager/project/README.md

```diff
@@ -26,7 +28,59 @@ The `components.specs` block contains maps of project configuration, with the fo
 | `auto_create_network` | boolean |  false   | automatically create a default network in the Google project                                            |                   none                    |
 | `project_iam`         |  list   |  false   | List of IAM role bindings used to create IAM policy for the project (see details [below](#project-iam)) |                   none                    |
 
-### Project IAM
+### Parent configuration  
+The parent of the project must be either a folder or an organization, and can be configured by the `parent_folder`/`parent_org` variables, as well as in the MCCF file.  
+If these values are set in multiple places, the order of priority is:
+1. Variables set in module block. The `parent_folder` or `parent_org` variable will take priority over any MCCF configuration,
+   and will apply to all the projects configured in the MCCF file.   
+   This means that setting the parents for individual projects (i.e. priority 2) cannot be done when using variables to configure parent. 
+2. MCCF file: `components.specs.<project>`. If `folder_id` or `org_id` is set in the configuration for the project, 
+    that project will ignore any parent configuration configured in `components.common`
+3. MCCF file: `components.common`. The `folder_id` or `org_id` configured here will apply to all projects in the MCCF file,
+    provided there is no configuration from 1 or 2.
+
+> **NOTE**: At each priority level (i.e. 1, 2 or 3) there should only be configuration for **either** folder **OR** organization.  
```

Comment:
We should also clarify that folder takes precedence over org; or will it fail if both are set?